docroot/modules/custom/sitenow_articles/sitenow_articles.module

```diff
@@ -206,6 +206,23 @@ function sitenow_articles_preprocess_node(&$variables) {
             }
           }
         }
+        if (in_array($variables['view_mode'], ['default', 'full'])) {
+          // Check if we don't have any gallery images.
+          if (empty($node->get('field_gallery_images')->getValue())) {
```

Comment:
This logic will WSOD during deployment for any traffic to articles.

To reproduce, checkout main, sync sandbox, checkout this branch, uli and try to hit https://sandbox.uiowa.ddev.site/news/2020/11/blue-flower-now-listed-our-site

Reply:
Ah, speaking of defensive coding practices...

Reply:
Gonna take a look at this.

Reply:
Should avoid the WSOD now.